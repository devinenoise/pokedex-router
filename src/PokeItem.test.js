import React from "react";
import PokeItem from "./PokeItem.js";
import renderer from "react-test-renderer";

test('renders Detail.js correctly', () => {
  const tree = renderer
    .create(<PokeItem pokemon={[]}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});