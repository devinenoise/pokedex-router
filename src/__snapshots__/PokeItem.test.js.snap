// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Detail.js correctly 1`] = `
<div
  className="container"
>
  <h3
    className="creature"
  />
  <p
    className="type"
  >
    <strong>
      Type
    </strong>
    : 
     / 
  </p>
  <p
    className="attributes"
  >
    <strong>
      Height
    </strong>
    : 
    "
  </p>
  <p
    className="attributes"
  >
    <strong>
      Weight
    </strong>
    : 
     lbs
  </p>
  <p
    className="attributes"
  >
    <strong>
      HP
    </strong>
    : 
  </p>
  <img />
</div>
`;
